source/features/sort-conversations-by-update-time.tsx

```diff
@@ -31,5 +36,12 @@ function init(): void {
 }
 
 void features.add(import.meta.url, {
+	deduplicate: 'has-rgh-inner',
```

Comment:
Fixes deduplication issue (e.g. when selecting several filters one after the other).